plugins/package-managers/bazel/src/main/kotlin/Bazel.kt

```diff
@@ -143,6 +146,44 @@ class Bazel(
         )
     }
 
+    /**
+     * Collect the local path overrides from the `MODULE.bazel` file in [workingDir]. The overrides are collected with
```

Comment:
Note to myself: So far we mostly use quotes to refer to file names in docs, but we probably should consistently migrate to backticks instead.